components/button/button-toggle.js

```diff
@@ -2,6 +2,7 @@ import { css, html, LitElement } from 'lit';
 
 /**
  * A button container component for button toggles.
+ * @fires click - Internal event
```

Comment:
It doesn't seem possible to `@ignore` our new click event since it's assigned to a variable, so this is our hack for not showing it in the docs (see [this](https://github.com/BrightspaceUI/documentation/blob/main/pages/about/adding-component.md#component-code))